arch/arm/core/cortex_m/mpu/nxp_mpu.c

```diff
@@ -382,14 +540,14 @@ int arm_core_mpu_buffer_validate(void *addr, size_t size, int write)
 {
 	u8_t r_index;
 
-	/* Iterate all MPU regions */
-	for (r_index = 0U; r_index < _get_num_usable_regions(); r_index++) {
+	/* Iterate through all MPU regions */
+	for (r_index = 0U; r_index < _get_num_regions(); r_index++) {
 		if (!_is_enabled_region(r_index) ||
-		    !_is_in_region(r_index, (u32_t)addr, size)) {
+				!_is_in_region(r_index, (u32_t)addr, size)) {
```

Comment:
typical style is to line up continuations with the opening parenthesis, rather this wasn't changed

Reply:
Fixing this back, thanks.

(addressed)